Packages/com.bci4kids.bciessentials/Runtime/Scripts/Behaviors/P300ControllerBehavior.cs

```diff
@@ -25,31 +28,39 @@ public class P300ControllerBehavior : BCIControllerBehavior
         public float onTime = 0.2f;
         public float offTime = 0.3f;
 
-        [Header("Stimulus Flash Paradigm")]
+        [Header("Stimulus Flash Paradigms")]
+        [Header("Single Flash Properties")]
         [Tooltip("If true, only one SPO will flash at a time")]
         public bool singleFlash = true;
+
+        [Tooltip("If true, enables context-aware SPO single flashing")]
+        public bool contextAwareSingleFlash = false;
+
+        [Header("Multi Flash Properties")]
         [Tooltip("If true, enables multiple SPOs to flash at the same time." +
         "Needs to be true for rowColumn and checkerboard to work")]
         public bool multiFlash = false;
```

Comment:
Would it be better if these are a list or similar? I don't think you should be able to enable multiple of this at the same time, right?